app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee.mdx

```diff
@@ -0,0 +1,277 @@
+---
+meta:
+  title: What to Expect in Virtual Coffee
+  description: Things happen in Virtual Coffee (with how-to instructions).
+hero:
+  Hero: UndrawRemoteMeeting
+tags: [memberresources]
+order: 2
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+import { Link } from '@remix-run/react';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+We get many questions like "What to expect once we join Virtual Coffee?" "Where's the link to today's Coffee?" "How to join a Virtual Coffee event?" This article serves as a guide to answer these questions!
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## Coffees (Virtual Coffee Weekly Zoom Chats)
+
+<LeadText>
+
+We have Coffee on **Tuesdays at 9:00 AM ET** and **Thursdays at 12:00 PM ET** that last an hour. This is a good place and time to meet and get to know other members, old and new, through casual conversations.
+
+</LeadText>
+
+### How It Works
+
+After some pre-coffee chatter, our MC will make an intro with some background and important announcements while people filter in. Then we'll break the crowd into breakout rooms for more relaxed and intimate conversations.
+
+When we move to the small group conversations, there will be a <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary#room-leader">Room Leader</Link> and <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary#note-taker">Note Taker</Link> for the room. They'll explain how we do introductions and how to participate. We value our members and the ways they're most comfortable participating.
+
+If you're here to listen and chill? No problem! If you can't turn on your camera or aren't comfortable having your camera or even sound on? Feel free to turn them off! There is no obligation for you to have them on. We want you to be as comfortable as possible in our Coffees.
+
+We drop the schedule in the `#announcement` channel on Slack every Monday and the day of the Coffee. There will also be an announcement to join the Coffee in the `#vc-events` channel 10 minutes before the Coffee start, along with the link to join.
+
+### Joining Coffees
+
+- Head over to the `#vc-events` channel on Slack.
+- Click the "Join Event" button to join the Coffee.
+
+Now you will be directed to the Coffee's Zoom.
+
+</TextContainer>
+
+<TextContainer>
+
+## Co-Working with Other Members
+
+<LeadText>
+
+Do you want to work or learn with people around, need help and a pair program, or only want to hang out with other members? Thanks to **Dan Ott**, we have a `#co-working-room` channel linked to our Zoom, so our members can connect anytime! It's open 24/7!
```

Comment:
Link to Dan's profile on the members page maybe?

Reply:
We don't have link to each member on members page.
But we can link his GitHub here.

Which make me think, should we link all the members name appear in the resources to their GitHub?
I did that previously in [Guide To Asking Questions About Your Code](https://virtualcoffee.io/resources/virtual-coffee/coding-questions-guide), but then I took them out.

Reply:
🤔 I was thinking about the link to the maintainers, but he's not all of the maintainers. 😆 

Good idea, but could be its own PR.

Reply:
There's also no links to maintainers only 😅

Reply:
Resolved this by adding a link to the [blog post](https://danott.dev/thoughts/virtual-coffee-coworking) 😁